geoh5py/objects/surveys/electromagnetics/base.py

```diff
@@ -466,6 +466,11 @@ def receivers(self, receivers: BaseEMSurvey):
         if receivers.tx_id_property is not None:
             self.edit_em_metadata({"Tx ID property": receivers.tx_id_property.uid})
 
+            if isinstance(
+                self.tx_id_property, ReferencedData | IntegerData
+            ) and isinstance(receivers.tx_id_property, ReferencedData | IntegerData):
```

Comment:
Question: why when IntegerData?

Reply:
Same as for any data, if they share type, they share visual parameters.

Reply:
Then why resricting it to IntergerData and ReferencedData only?

Reply:
Enforced by GA